style.css

```diff
@@ -290,3 +310,149 @@ fieldset textarea {
   font-family: sans-serif;
   margin-bottom: 3rem;
 }
+
+.work-space-1 {
+  flex-direction: column;
+}
+
+.lets-chat {
+  color: #9390ff;
+  margin: 1rem 1rem 2.5rem 1rem;
+  margin-top: 6rem;
+  font-size: 3rem;
+}
+
+.icons-and-email {
+  flex-direction: column;
+}
+
+@media screen and (max-width: 800px) {
+  .nav-ul,
+  .arrow-1,
+  .lets-chat {
+    display: none;
+  }
+}
+
+/* Media Querie */
+@media screen and (min-width: 800px) {
```

Comment:
- [ ] Please, uses media queries for the Desktop breakpoints in the CSS file (min-width > 992px - 1100px)
- Hint: `@media screen and (min-width: 1000)` (something like this) 🐶